samples/bluetooth/le_periph_pm/boards/alif_b1_pm.dtsi

```diff
@@ -28,7 +28,7 @@
 
 	power-states {
 		off: off {
-			min-residency-us = < 100000 >;
+			min-residency-us = < 4000 >;
```

Comment:
Wouldn't hurt to have a comment explaining on what this value is based on. If possible.